moneymanager/__main__.py

```diff
@@ -23,10 +23,9 @@
                 category_column: 'object',
                 source_column: 'object'
     }
-    CsvManager.preprocess_csv_files(input_directory)
-    transactions = CsvManager.merge_csv_files(input_directory, dtypes)
+    transactions = CsvManager.get_transaction_data(input_directory, dtypes)
```

Comment:
This is a better strategy than changing the source files, as was previously the case